packages/@webex/plugin-meetings/src/reachability/index.ts

```diff
@@ -341,18 +341,21 @@ export default class Reachability {
       return Promise.resolve(results);
     }
 
-    // @ts-ignore
-    const includeTcpReachability = this.webex.config.meetings.experimental.enableTcpReachability;
-
     LoggerProxy.logger.log(
       `Reachability:index#performReachabilityChecks --> doing UDP${
-        includeTcpReachability ? ' and TCP' : ''
+        // @ts-ignore
+        this.webex.config.meetings.experimental.enableTcpReachability ? ' and TCP' : ''
```

Comment:
in the previous file you set `config.experimental.enableTcpReachability` but here you read `config.meetings.experimental.enableTcpReachability` how is this values copied?

Also, is the path always OK? Do not you need Optional chaining operator?

Reply:
good point, I tested it with the web app and it was working when I called _toggleTcpReachability() so it must be copied somewhere, but I'll check

I'll also add the optional chaining

Reply:
I've checked and webex.config.meetings is the same as this.config in Meetings class:
![image](https://github.com/webex/webex-js-sdk/assets/7473332/fdd51cf7-94db-4833-b5f3-3765d2c64de6)

Reply:
as mentioned in the other comment, optional chaining is not needed